packages/manager/modules/pci/src/components/project/images-list/translations/Messages_fr_FR.json

```diff
@@ -1,6 +1,5 @@
 {
-
-  "pci_project_instances_available_only": "Masquer les options non disponibles",
+  "pci_project_instances_available_only": "Montrer les localisations indisponibles",
```

Comment:
nit: The key name should reflect as much as possible what it is written.
For example: pci_project_instances_display_nonavailable

Reply:
This is done